app/views/shared/xml/_activity.xml.haml

```diff
@@ -45,10 +45,10 @@
     - benefitting_countries_with_percentages(activity.benefitting_countries).each do |country|
       %recipient-country{"code" => country.code, "percentage" => country.percentage}
         %narrative= country.name
-  - elsif activity.recipient_country?
+  - elsif activity.recipient_country? && !activity.is_ispf_funded?
```

Comment:
This is added on each `elsif` so that we don't need to nest the conditionals any deeper.

Reply:
I'm of two minds about this. Because these legacy fields cannot be populated for new activities, neither via the UI nor via bulk import, they'll always be nil for ISPF activities, so the check is not necessary. It's not harmful either, since the first condition will always evaluate to `false`. (As long as ruby keeps doing "short circuit" evaluation, and we'd have bigger problems if this changed...)

Reply:
Understood. I'll remove these!